docs/src/reference/asciidoc/appendix/release-notes/7.14.0.adoc

```diff
@@ -1,7 +1,21 @@
 [[eshadoop-7.14.0]]
 == Elasticsearch for Apache Hadoop version 7.14.0
 
+[[known-issues-7.14.0]]
```

Comment:
Can you also add this to 7.14.1 ?
https://www.elastic.co/guide/en/elasticsearch/hadoop/current/eshadoop-7.14.1.html

Reply:
Done.